src/dosbox.cpp

```diff
@@ -834,8 +834,19 @@ void DOSBOX_Init()
 	        "   4: 2 MB for the FBI and one TMU with 2 MB (default).\n"
 	        "  12: 4 MB for the FBI and two TMUs, each with 4 MB.");
 
-	pbool = secprop->Add_bool("voodoo_multithreading", only_at_start, true);
-	pbool->Set_help("Use threads to improve 3dfx Voodoo performance (enabled by default).");
+	// Deprecate the boolean Voodoo multithreading setting
+	pbool = secprop->Add_bool("voodoo_multithreading", deprecated, false);
+	pbool->Set_help("Use 'voodoo_threads = auto / <value>'.");
```

Comment:
Not bad, but for consistency I'd suggest to use simply this:

``Renamed to 'voodoo_threads'``

Then we don't need to duplicate information in the deprecation text (one less thing to get out of sync).

<img width="915" alt="image" src="https://github.com/user-attachments/assets/72c95256-e580-4734-aca7-8c1415960823">

Reply:
Ok!